Services/UserService/Validators/EditUserPasswordValidator.cs

```diff
@@ -0,0 +1,21 @@
+﻿using DTO.RestRequests;
+using FluentValidation;
+
+namespace UserService.Validators
+{
+    public class EditUserPasswordValidator: AbstractValidator<EditUserRequest>
+    {
+        public EditUserPasswordValidator()
+        {
+            RuleFor(user => user.OldPassword)
+                .NotEmpty()
+                .WithMessage(ErrorsMessages.EmptyPassword);
+            RuleFor(user => user.NewPassword)
+                .NotEmpty()
+                .WithMessage(ErrorsMessages.EmptyPassword);
+            RuleFor(user => user.OldPassword)
```

Comment:
Why you have divided rules for old password in 2 parts? * *confused noises* * =)

Reply:
Sorry, don't get you.  It is two different parts with old and new password

Reply:
> Sorry, don't get you. It is two different parts with old and new password

Why not like that?

            RuleFor(user => user.NewPassword)
                .NotEmpty()
                .WithMessage(ErrorsMessages.EmptyPassword);
            RuleFor(user => user.OldPassword)
                .NotEmpty()
                .WithMessage(ErrorsMessages.EmptyPassword);
                .NotEqual(user => user.NewPassword)
                .WithMessage(ErrorsMessages.OldAndNewPasswordsEqual);

Reply:
I think it is a little different 
We have separate rules for each properties and one rule for their combination